src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Rendering/EventForDotNet.ts

```diff
@@ -28,12 +34,15 @@ function isCheckbox(element: Element | null) {
 
 // The following interfaces must be kept in sync with the UIEventArgs C# classes
 
-type EventArgsType = 'mouse' | 'keyboard' | 'change' | 'unknown';
+type EventArgsType = 'mouse' | 'keyboard' | 'change' | 'focus' | 'unknown';
 
 export interface UIEventArgs {
   Type: string;
 }
 
+export interface UIFocusEventArgs extends UIEventArgs {
```

Comment:
I don't think this interface needs to be exported, as we're only using the type parameters within this file as a way of checking that the constructor args are of a recognized format.

It wouldn't be *harmful* to export it either, but if we are doing so, it would be good to export all the others too for consistency.

Reply:
Ah right, I didn't notice that. Will undo the export until we actually need these types exported